### DISCUSSION TITLE
Tracing does not give all the information?

### DISCUSSION THREAD

**Discussion by JakubKoralewski** [prisma/prisma#16966] [Q&A]
Here is a get request trace from Jaeger:
![obraz](https://user-images.githubusercontent.com/43069023/208980523-5fa9974f-837d-41f3-a2a7-af87e0c39676.png)
Prisma query seems to be taking 574ms. It's a findMany of posts. However, the only accounted subtraces cover the first few milliseconds. Are the `db_query`ies just covering the creation of the query and not the actual execution? Or does the database really return from the query in less than 1ms?

How do I find out what is causing Prisma to not return for the remaining 500ms? I guess this is a feature request/bug report for the tracing feature? 

Don't be fully alarmed by the very long time of the request, I picked the slowest query of my stress tests. Please stay half-alarmed only.

I'd appreciate any help :) 🎅

**Comment by janpio** [comment]
That is weird and unexpected - most probably a bug in our implementation.
Can you consistently reproduce this?
Is you application maybe simple enough to put this into a reproduction you can share with us in a bug issue?

**Reply by JakubKoralewski** [reply]
![obraz](https://user-images.githubusercontent.com/43069023/208995673-7fe1602d-ad92-408b-8db3-6bb13e0796bd.png)
![obraz](https://user-images.githubusercontent.com/43069023/208995747-28287325-02e4-42c1-94d1-60dec0873b14.png)

Seems reproducible. I'd need to make a separate repro unfortunately.

**Reply by JakubKoralewski** [reply]
@janpio created a bug issue https://github.com/prisma/prisma/issues/16980

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
